project_cost_report/static/src/js/report_action.js

```diff
@@ -0,0 +1,386 @@
+/*
+    © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+odoo.define('project_cost_report.ReportAction', function (require) {
+'use strict';
+
+var core = require('web.core');
+var session = require('web.session');
+var Widget = require('web.Widget');
+var ControlPanelMixin = require('web.ControlPanelMixin');
+var session = require('web.session');
+var framework = require('web.framework');
+var crash_manager = require('web.crash_manager');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'crash_manager' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3675949)